docroot/themes/humsci/humsci_basic/patterns/structured-card/structured-card.html.twig

```diff
@@ -18,7 +18,9 @@
 
   <div class="hb-card__content">
     {% if title %}
-      <div class="hb-card__title">{{ title }}</div>
+      <div class="hb-card__title">
```

Comment:
so the structured card doesnt have an arrow right?

Reply:
Correct. The structured card is used for `people` which does not have an arrow.